src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
-import { take, takeUntil } from 'rxjs/operators';
+import { take, takeUntil, distinctUntilChanged } from 'rxjs/operators';
```

Comment:
remove distinctUntilChanged, you don't use it

Reply:
fixed